### PR TITLE
Update Spring 2023 Board

### LINKED ISSUE #752: Spring 2023 term update
### What's your idea?

The webmaster must accurately update `src/lib/public/board/data/officers.json` to update the Spring 2023 board members.

### TODO

- [ ] Prepend `Term.Spring23` to [`VISIBLE_TERMS`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/c0c027598fd32852e097b9ca98de0ac99c1c6b74/src/lib/public/board/data/index.ts#L6)
- [ ] Provide baseline update for [`src/lib/public/board/data/officers.json`](https://github.com/EthanThatOneKid/acmcsuf.com/blob/c0c027598fd32852e097b9ca98de0ac99c1c6b74/src/lib/public/board/data/officers.json) entries

### PR DESCRIPTION
**Pull request by k4rni**
## Need to complete:
- [x] Add images/socials to board members
- [x] Fix tier list
- [x] Fix colors
- [x] Add youtube social

Fix #752.

### CODE REVIEW

File: src/lib/public/board/types.ts [hunk 1]
```diff
@@ -33,6 +33,7 @@ export enum Social {
   Discord = 'discord',
   LinkedIn = 'linkedin',
   Instagram = 'instagram',
+  Youtube = 'youtube',
```
Comment (EthanThatOneKid) [inline src/lib/public/board/types.ts]: Nit: YouTube has a capital T.
```suggestion
  YouTube = 'youtube',
```

File: src/routes/(site)/about/officer-profile.svelte [hunk 2]
```diff
@@ -22,21 +23,55 @@
...
-    AI: 'acm-emerald',
-    Marketing: 'acm-red',
-    'Special Events': 'acm-lemon',
-    nodebuds: 'brand-header',
-  });
+  /**
+   * teamColors maps team names to CSS color variables. We call Object.entries on
+   * it to make it easier to iterate and search.
+   */
+  const teamColors = {
```
Comment (EthanThatOneKid) [inline src/routes/(site)/about/officer-profile.svelte]: Nit: Rename `teamColors` to `COLORS` and move to `src/lib/public/board/data/colors.ts`. Export from `src/lib/public/board/data/index.ts`.

File: src/routes/(site)/about/officer-profile.svelte [hunk 3]
```diff
@@ -161,6 +204,17 @@
...
                 </p>
               {/if}
+              {#if officerSocials.youtube}
+                <p class="officer-youtube">
+                  <a
+                    target="blank"
+                    title={officerSocials.youtube}
+                    href="https://www.youtube.com/@{officerSocials.youtube}"
+                  >
+                    <BwIcon src="/assets/svg/youtube.svg" alt="Youtube icon" /><span>Youtube</span>
```
Comment (EthanThatOneKid) [inline src/routes/(site)/about/officer-profile.svelte]: Nit: YouTube has a capital T.
```suggestion
                    <BwIcon src="/assets/svg/youtube.svg" alt="YouTube icon" /><span>YouTube</span>
```

File: src/lib/public/board/data/tiers.json
Comment (EthanThatOneKid) [inline src/lib/public/board/data/tiers.json:1]: I do not see any mistakes in this file 👍

File: src/lib/public/board/types.ts [hunk 4]
```diff
@@ -53,7 +54,8 @@ interface Position {
 
 export interface Officer {
   fullName: string;
-  picture: string;
+  picture?: string;
+  legacyPicture?: string;
```
Comment (EthanThatOneKid) [inline src/lib/public/board/types.ts:65]: Please leave a JSDoc comment explaining the `{#if}` mechanism that conditionally renders the officer picture.

File: static/global.css
Comment (EthanThatOneKid) [inline static/global.css:1]: These `static/global.css` changes look great to me!

**Review by EthanThatOneKid — COMMENTED** [review]
It seems like you've gone ahead and found board pictures for a lot of the new set of board members; fantastic! I have taken a look at your PR and submitted my initial review.

**Comment by EthanThatOneKid** [comment]
### Spectacular progress screenshot

Screenshot

**Review by EthanThatOneKid — APPROVED** [review]
These are very welcome changes; LGTM! :godmode:

**Comment by jaasonw** [comment]
i just think it's a little bit weird the old ppl still have blue as their background highlight

**Comment by k4rni** [comment]
Since those were manually created we can't remove the color.

**Comment by jaasonw** [comment]
with enough time in photoshop u could but yea ain't no one down for that shit

**Pull request merged**